src/workflows-app/components/FindWorkflowModal.js

```diff
@@ -1,16 +1,158 @@
+    method_source: 'GitHub',
+    method_version: 'Optimus_v5.7.2',
+    method_url: 'https://raw.githubusercontent.com/broadinstitute/warp/Optimus_v5.7.2/pipelines/skylab/optimus/Optimus.wdl',
+  },
+  {
+    method_name: 'MultiSampleSmartSeq2SingleNucleus',
+    method_description: 'The MultiSampleSmartSeq2SingleNucleus pipeline runs multiple snSS2 samples in a single pipeline invocation.',
+    method_source: 'GitHub',
+    method_version: 'MultiSampleSmartSeq2SingleNucleus_v1.2.18',
+    method_url:
+      'https://raw.githubusercontent.com/broadinstitute/warp/MultiSampleSmartSeq2SingleNucleus_v1.2.18/pipelines/skylab/smartseq2_single_nucleus_multisample/MultiSampleSmartSeq2SingleNucleus.wdl',
+  },
+  {
+    method_name: 'scATAC',
+    method_description: 'Processing of single-cell ATAC-seq data with the scATAC pipeline.',
+    method_source: 'GitHub',
+    method_version: 'scATAC_v1.3.1',
+    method_url: 'https://raw.githubusercontent.com/broadinstitute/warp/scATAC_v1.3.1/pipelines/skylab/scATAC/scATAC.wdl',
+  },
+  {
+    method_name: 'WholeGenomeGermlineSingleSample',
+    method_description: 'Processes germline whole genome sequencing data.',
+    method_source: 'GitHub',
+    method_version: 'WholeGenomeGermlineSingleSample_v3.1.10',
+    method_url:
+      'https://raw.githubusercontent.com/broadinstitute/warp/WholeGenomeGermlineSingleSample_v3.1.10/pipelines/broad/dna_seq/germline/single_sample/wgs/WholeGenomeGermlineSingleSample.wdl',
+  },
+  {
+    method_name: 'ExomeGermlineSingleSample',
+    method_description: 'Processes germline exome/targeted sequencing data.',
+    method_source: 'GitHub',
+    method_version: 'ExomeGermlineSingleSample_v3.1.9',
+    method_url:
+      'https://raw.githubusercontent.com/broadinstitute/warp/ExomeGermlineSingleSample_v3.1.9/pipelines/broad/dna_seq/germline/single_sample/exome/ExomeGermlineSingleSample.wdl',
+  },
+];
+
+const FindWorkflowModal = ({ onDismiss, workspace }) => {
+  const [selectedSubHeader, setSelectedSubHeader] = useState('browse-suggested-workflows');
+  const [loading, setLoading] = useState(false);
+
+  const signal = useCancellation();
+
+  const subHeadersMap = {
+    'browse-suggested-workflows': 'Browse Suggested Workflows',
+    ...(getConfig().isURLEnabled && { 'add-a-workflow-link': 'Add a Workflow Link' }),
+    ...(getConfig().isDockstoreEnabled && { 'go-to-dockstore': h(TextCell, {}, ['Dockstore', icon('export', { style: { marginLeft: '0.5rem' } })]) }),
+  };
+
+  const isSubHeaderActive = (subHeader) => selectedSubHeader === subHeader;
 
-const FindWorkflowModal = ({ onDismiss }) => {
   return h(
     ModalDrawer,
     {
-      'aria-label': 'Find a Workflow Modal',
+      'aria-label': 'find-workflow-modal',
```

Comment:
We should leave this label the same as per [this comment](https://github.com/DataBiosphere/terra-ui/pull/3986#discussion_r1244289288)